src/index.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html lang="en">
+
```

Comment:
No need to add empty line between parent and children tag, only between siblings